src/components/search/ExactSwitch.tsx

```diff
@@ -0,0 +1,39 @@
+import { Stack, Switch, Typography } from '@mui/material'
+import { Fragment, useState } from 'react'
+interface SimpleSearchExactSwitch {
```

Comment:
```suggestion
interface ExactSwithProps {
```
Let's always use the name of the component and add "Props" suffix.
Also please add line breaks before and after type definition.